### PR TITLE
calc: added html entry in downloadas menu

### PR DESCRIPTION
**Pull request by Rash419**
Change-Id: Icd816a326bc0264400f375444cf2e165d8f2e9ac

* Resolves: # 
* Target version: master

### CODE REVIEW

**Review by meven — APPROVED** [review]

**Comment by Rash419** [comment]
I will wait for @pedropintosilva's approval because I have added a image : )

**Comment by meven** [comment]
I hope it does not take up too much space.

**Pull request merged**
